research_spec.md

```diff
@@ -36,7 +36,9 @@ For a round that will start its active phase at time `t`, the round proceeds lik
 
 - At any time between t-`AuctionClosingSeconds` and t, the autonomous auctioneer closes the auction by submitting a transaction to the auction contract, as described above. The autonomous auctioneer is trusted to follow the instructions specified above. This may designate a party as the express lane controller for the round, and if so it will deduct funds from that party's deposit account and distribute or burn those funds.
 
-- At any time between the designation of the express lane controller and the end of the round (t+`RoundDurationSeconds`), the express lane controller can send a transaction to the auction contract to transfer the express lane controller right for the round to a different address. By doing this, the giver loses all of its own rights in the round.
+- At any time between the auction being resolved for the round starting at t and the end of that round (t+`RoundDurationSeconds`), the express lane controller, or an alternative `transferor` if set, can call the `transferExpressLaneController` on the auction contract to transfer the express lane controller right for the round to a new address. By calling `transferExpressLaneController` and transferring express lane controller rights the current controller loses all of its own rights in the round. The transferee receives express lane controller rights from the moment they are transferred, as well as the right to transfer to another address, either directly or by an alternative transferor if set. Upon observing a transfer the sequencer resets the round sequence number to 0. 
+  
+  Each express lane controller can set their own alternative `transferor` by calling `setTransferor` function at any time, this `transferor` will have the ability to transfer the rights of any current or future round controlled by that express lane controller. Along with the new transferor address, an optional round number is provided when calling `setTransferor` which specifies when the transferor can next be updated. If a round number is not provided the transferor can be updated at any time.
```

Comment:
Presumably the transferror for round `i` can't say that the transferror can next be updated in round `i+100`, right?  Doesn't there need to be some limit on this, so a controller can't block transfers by a later controller?

Reply:
Each bidder can set their own transferror. So the limit they impose is only on themselves, not on any other controller.

Reply:
Yes, that makes sense. But I think the spec should say it explicitly.

Reply:
Updated